src/__myproject__/_orchestration/clickhouse_export.py

```diff
@@ -0,0 +1,110 @@
+# Databricks notebook source
+# MAGIC %run ../app/bootstrap
+
+# COMMAND ----------
+
+from pyspark.sql import utils, SparkSession, DataFrame
+from pyspark.sql import functions as f
+from pyspark.dbutils import DBUtils
+from clickhouse_driver import Client as ClickhouseClient
+from logging import Logger
+import daipe as dp
+
+# COMMAND ----------
+
+entity = dp.fs.get_entity()
+
+# COMMAND ----------
+
+@dp.notebook_function()
+def init_widgets(widgets: dp.Widgets):
+    widgets.add_text("config_url", "", "Config url")
+
+# COMMAND ----------
+
+@dp.notebook_function(dp.get_widget_value("config_url"))
+def load_config(config_url: str, logger: Logger, dbutils: DBUtils, spark: SparkSession):
+    try:
+        config = spark.read.json(config_url).collect()[0].asDict()
```

Comment:
maybe use daipe [read_json](https://www.notion.so/datasentics/Decorator-functions-e6f89bfd2c49473f8fde8bf25f6580bd#efa2237730664d3b922c4798b46dcf30)?

Reply:
I want to load the json as a dict and found dp.read_json to be restrictive since it can only be used in a transformation.